### PR TITLE
Add documentation for AIX and Intune parser

### PR DESCRIPTION
**Pull request by Thibault-MAL**
Please find my latest contribution to the wonderful docs.sekoia.io

### CODE REVIEW

**Review by CharlesLR-sekoia — APPROVED** [review]

File: docs/xdr/features/collect/integrations/endpoint/ibm_aix.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+In this guide, you will configure the gateway to forward events to syslog.
+
+### Prerequisites
+
+An internal log concentrator (Rsyslog) is required to collect and forward events to SEKOIA.IO.
+
+### Enable Syslog forwarding
+
+Log on your NIOS appliance and follow [this guide](https://www.ibm.com/docs/en/dsm?topic=aado-configuring-aix-audit-dsm-send-syslog-events-qradar) to enable syslog forwarding
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/endpoint/ibm_aix.md:21]: NIOS is another technology. I suggest deleting this part of the sentence.
```suggestion
Follow [this guide](https://www.ibm.com/docs/en/dsm?topic=aado-configuring-aix-audit-dsm-send-syslog-events-qradar) to enable syslog forwarding for audit events
```

File: docs/xdr/features/collect/integrations/endpoint/microsoft_intune.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+To forward events to SEKOIA.IO, create [an Event hub namespace and an Event hub](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create) with a consumergroup.
+
+Once created, in your EventHubs, go to `Setting > Shared access policies`.
+Create a new policy with the option `Listen` then copy the `Connection string-primary key`.
+
+Create a [Storage accounts](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create) or use an existing one. Go to `Data storage > containers` and create a new container.
+Then go to `Security + networking > Access keys` and copy the key1 `Connection string`
+
+Configure Azure Front door to stream its logs to the EventHub with [this guide](https://docs.microsoft.com/en-us/azure/frontdoor/standard-premium/how-to-logs#configure-logs).
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/endpoint/microsoft_intune.md:25]:
```suggestion
Configure Microsoft Intune door to stream its logs to the EventHub with [this guide](https://learn.microsoft.com/en-us/mem/intune/fundamentals/review-logs-using-azure-monitor).
```

**Review by squioc — CHANGES_REQUESTED** [review]

File: docs/xdr/features/collect/integrations/endpoint/ibm_aix.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+uuid: e8cb3372-41c8-409f-8e0e-ca4e6cecf7e6
+name: IBM AIX 
+type: intake
+
+## Overview
+
+AIX (Advanced Interactive eXecutive), is a series of proprietary Unix operating systems developed and sold by IBM for several of its computer platforms.
+
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/endpoint/ibm_aix.md:8]: Recent change on the documentation, to associate rule to datasource, we should include a new autogenerated document
```suggestion
{!_shared_content/operations_center/detection/generated/suggested_rules_e8cb3372-41c8-409f-8e0e-ca4e6cecf7e6_do_not_edit_manually.md!}

```

File: docs/xdr/features/collect/integrations/endpoint/microsoft_intune.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+uuid: c20528c1-621e-4959-83ba-652eca2e8ed0
+name: Microsoft Intune
+type: intake
+
+## Overview
+
+Microsoft Intune helps you protect your workforce's corporate data by managing devices and apps. Intune provides mobile device management (MDM) and mobile app management (MAM) from a secure cloud-based service that is administered using the Microsoft Endpoint Manager admin center. Using Intune, you ensure your workforce's corporate resources (data, devices, and apps) are correctly configured, accessed, and updated, meeting your company's compliance policies and requirements.
+
+This setup guide describe how to forward events produced by `Microsoft Intune` to SEKOIA.IO.
+
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/endpoint/microsoft_intune.md:10]: Recent change on the documentation, to associate rule to datasource, we should include a new autogenerated document
```suggestion
{!_shared_content/operations_center/detection/generated/suggested_rules_c20528c1-621e-4959-83ba-652eca2e8ed0_do_not_edit_manually.md!}

```

**Review by squioc — CHANGES_REQUESTED** [review]

**Comment by squioc** [comment]
My bad. Could you add these two new documentations in the mkdocs.yml (in order to display them in the menu)?

File: docs/xdr/features/collect/integrations/endpoint/ibm_aix.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+### Enable Syslog forwarding
+
+Log on your NIOS appliance and follow [this guide](https://www.ibm.com/docs/en/dsm?topic=aado-configuring-aix-audit-dsm-send-syslog-events-qradar) to enable syslog forwarding
+
+## Create the intake
+
+Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the format IBM AIX.
+
+
+## Transport to SEKOIA.IO
```
Comment (ka0ula) [inline docs/xdr/features/collect/integrations/endpoint/ibm_aix.md:28]:
```suggestion
## Send logs to SEKOIA.IO
```

**Review by ka0ula — CHANGES_REQUESTED** [review]

File: docs/xdr/features/collect/integrations/endpoint/microsoft_intune.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+### Prerequisites
+
+To forward events to SEKOIA.IO, create [an Event hub namespace and an Event hub](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create) with a consumergroup.
+
+Once created, in your EventHubs, go to `Setting > Shared access policies`.
+Create a new policy with the option `Listen` then copy the `Connection string-primary key`.
+
+Create a [Storage accounts](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create) or use an existing one. Go to `Data storage > containers` and create a new container.
+Then go to `Security + networking > Access keys` and copy the key1 `Connection string`
```
Comment (ka0ula) [inline docs/xdr/features/collect/integrations/endpoint/microsoft_intune.md:24]:
```suggestion
Then go to `Security + networking > Access keys` and copy the key1 `Connection string`.
```

**Pull request merged**